src/ast/mod.rs

```diff
@@ -380,7 +380,10 @@ pub enum Expr {
         right: Box<Expr>,
     },
     /// CompositeAccess (postgres) eg: SELECT (information_schema._pg_expandarray(array['i','i'])).n
-    CompositeAccess { expr: Box<Expr>, key: Ident },
+    CompositeAccess {
```

Comment:
I wonder why this was reformatted. Maybe you are using a different version of `cargo fmt` or something 🤔